transport/src/main/java/io/netty/channel/internal/svm/NioEventLoopSubstitution.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.channel.internal.svm;
+
+import java.util.Queue;
+import java.util.concurrent.LinkedBlockingDeque;
+
+import com.oracle.svm.core.annotate.Substitute;
+import com.oracle.svm.core.annotate.TargetClass;
+
+@TargetClass(className = "io.netty.channel.nio.NioEventLoop")
```

Comment:
Also what about the other `EventLoop` implementations ?

Reply:
It addresses the static factory method on `io.netty.channel.nio.NioEventLoop`.

Reply:
basically I was asking why we not need this in `EpollEventLoop` as well.

Reply:
I see… and wonder if the native epoll event loop would be usable at all with Graal native over JNI. I don't the answer to that. Probably not without more effort.

Reply:
got it... I guess we can do this as a followup